src/util/Util.js

```
@@ -46,6 +46,26 @@ class Util {
   }
 
   /**
+   * Format a string
+   * @param {string} format one of `bold`, `italic`, `underlined`, `strikethrough`
+   * @param {string} str String to format
+   * @returns {string} formatted string
+   */
+  static applyFormat(format, str) {
+    const formatters = {
+      bold: '**',
+      italic: '_',
+      underlined: '__',
+      strikethrough: '~~',
+    };
+    if (!(format in formatters)) return str;
```

Should probably just throw an error.